### PR TITLE
Add Python path to be able to do import qgis in a simple Python inter…

### PR DESCRIPTION
**Pull request by sbrunner**
…preter

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -192,7 +192,8 @@ ENV QGIS_SERVER_LOG_STDERR=1 \
     FCGID_IDLE_TIMEOUT=300 \
     FCGID_IO_TIMEOUT=40 \
     FILTER_ENV='' \
-    GET_ENV=env
+    GET_ENV=env \
+    PYTHONPATH=./usr/local/share/qgis/python/
```
Comment (arnaud-morvan) [inline Dockerfile:196]: Are you sure about the leading dot in path ?

Comment (sbrunner) [reply Dockerfile:196]: Oups, thanks :-)

Comment (arnaud-morvan) [reply Dockerfile]: Is this possible, maybe better ?
COPY --from=builder-server /usr/lib/python3/dist-packages /usr/lib/python3/dist-packages

Comment (danduk82) [reply Dockerfile]: maybe the full `/usr/lib/python3/` ?

Comment (sbrunner) [reply Dockerfile]: @arnaud-morvan 
In the official Docker image they do the same kind of things:
https://github.com/qgis/QGIS/blob/master/.docker/qgis.dockerfile#L79

Comment (arnaud-morvan) [reply Dockerfile]: Ok, note QGIS is in loaded from /usr/lib/python3/dist-packages/qgis

```bash
$ docker run --rm -ti qgis/qgis:release-3_16 python3
Python 3.8.10 (default, Sep 28 2021, 16:10:42) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import qgis
>>> qgis
<module 'qgis' from '/usr/lib/python3/dist-packages/qgis/__init__.py'>
```

**Review by arnaud-morvan — APPROVED** [review]

**Review by danduk82 — APPROVED** [review]

**Pull request merged**